packages/features/ee/workflows/pages/workflow.tsx

```diff
@@ -244,6 +264,11 @@ function WorkflowPage() {
                     workflowId={+workflowId}
                     selectedEventTypes={selectedEventTypes}
                     setSelectedEventTypes={setSelectedEventTypes}
+                    verifiedNumbers={verifiedNumbers?.map(
+                      (verifiedNumber: VerifiedNumber) => verifiedNumber.phoneNumber
+                    )}
+                    setNewVerifiedNumbers={setNewVerifiedNumbers}
+                    newVerifiedNumbers={newVerifiedNumbers}
```

Comment:
I'm getting rid of all the prop-drilling here. We can just use trpc directly on the children and invalidate on mutation. No need to keep local states in sync